tools/flang2/flang2exe/cgmain.cpp

```diff
@@ -1701,10 +1669,8 @@ schedule(void)
     if ((!XBIT(69, 0x100000)) && BIH_NODEPCHK(bih) &&
         (!ignore_simd_block(bih)) || XBIT(191, 0x4)) {
       fix_nodepchk_flag(bih);
```

Comment:
Is this used now?

Reply:
Yes, it is. The function `fix_nodepchk_flag` could use a better name, but the new `llvm.access.group` metadata mechanism still depends on the `BIH_NODEPCHK2` flag set by the function.